modules/apigee_m10n_teams/src/MonetizationTeams.php

```diff
@@ -172,7 +172,7 @@ public function fieldWidgetInfoAlter(array &$info) {
    */
   public function purchasedPlanAccess(EntityInterface $entity, $operation, AccountInterface $account) {
     /** @var \Drupal\apigee_m10n_teams\Entity\TeamsPurchasedPlanInterface $entity */
-    if ($entity->isTeamPurchasedPlan() && ($team = $entity->get('team')->entity)) {
+    if ($entity->isTeamPurchasedPlan() && ($team = $entity->getTeamEntity())) {
```

Comment:
I'm OK with adding `getTeamEntity()` but these should both do the same thing.

Reply:
This should work the same.